pkgs/build-support/setup-hooks/make-wrapper.sh

```diff
@@ -19,98 +19,98 @@
 #                                   are read first and used as VALS
 # --suffix-contents
 makeWrapper() {
-    local original=$1
-    local wrapper=$2
+    local original="$1"
+    local wrapper="$2"
```

Comment:
I suspected so because it was unquoted in most places here, but:
```
[abbradar@abbradarbook:~]$ test() { local a=a b c; echo $a; }

[abbradar@abbradarbook:~]$ test
a

[abbradar@abbradarbook:~]$ test() { local a="a b c"; echo $a; }

[abbradar@abbradarbook:~]$ test
a b c
```